loader/src/sources/albertsons/index.js

```diff
@@ -261,9 +277,12 @@ async function getData(states) {
       result.external_ids = getUniqueExternalIds(
         group.flatMap((l) => l.external_ids)
       );
-      result.availability.products = [
-        ...new Set(group.flatMap((l) => l.availability.products)),
+      const products = [
+        ...new Set(
+          group.flatMap((l) => l.availability.products).filter(Boolean)
+        ),
       ];
+      result.availability.products = products.length ? products : undefined;
```

Comment:
Between @cmroberts’s PR for H-E-B (https://github.com/usdigitalresponse/univaf/pull/452#discussion_r749841369) and this, I’m starting to think the “let’s send and show only `undefined` or a non-empty array” plan for products and doses was not really useful, and mainly results in extra work that isn’t helping anyone. :\